portx/modules/ROOT/pages/_partials/documents-view.adoc

```diff
@@ -1,51 +1,54 @@
 == Documents View
+[[img-documents-view, Documents View]]
+
+image::documents-view.png[img-documents-view, title="Documents View"]
+
 :icons: font
 The Documents View presents data on documents that are sent through xref:transaction-monitoring.adoc[Transaction Monitor].
-For more information on documents see xref:tracking-examples.adoc#tracking-documents[Tracking Examples].
+For more information on documents see xref:tracking-examples.adoc#tracking-documents[Tracking Documents].
 
 
 === Content
-Each row has 6 columns:
+Each row has six columns:
 
-. *From*: Who the document was sent from.
+From:: From whom the document was sent.
 
-. *To*: Who the document was sent to.
+To:: To whom the document was sent.
 
-. *Date*: Date and time at which the document was sent.
+Date:: Date and time at which the document was sent.
 
-. *Standard*: The standard used to send the document.
+Standard:: The standard used to send the document.
 
-. *Message Type*: The type of message that was sent.
+Message Type:: The type of message that was sent.
 
-. *Label*: Detail information describing the document.
+Label:: Detail information describing the document.
 
 === Document Detail Pane (Right)
 This pane is hidden when no document is selected. It contains:
 
-* Document ID icon:plus-square-o[role="blue"]
-* Document Definiton ID icon:plus-square-o[role="blue"]
-* Standard
-* Version
-* Message Type
-* From
-* To
-* Date
-* Duplicate Quiry
-* Message Payload icon:share[role="blue"]
-* Partnership ID icon:plus-square-o[role="blue"]
-
-==== Action Icons
-
-* The blue copy icon icon:plus-square-o[role="blue"] allows you to copy information to your clipboard
-* The blue share icon icon:share[role="blue"] opens a detail pane that shows the complete message payload.
+Document ID:: Unique copyable document identification.
+Document Definiton ID:: Unique copyable document definition identification. icon:plus-square-o[role="blue"]
+Standard:: The document standard used.
+Version:: The version of the document.
+Message Type:: The type/reason for the message.
+From:: From whom the document came.
+To:: To whom the document is going.
+Date:: Date the document was sent.
+Duplicate Quiry:: Indicating if a document is a duplicate.
+Message Payload:: icon:share[role="blue"] A link to the payload url and payload content
+Partnership ID:: icon:plus-square-o[role="blue"] Unique copyable partnership identification.
+
+==== *Detail Pane Action Icons*
+
+* The *blue copy icon* icon:plus-square-o[role="blue"] enables you to copy information to your clipboard
```

Comment:
Add a period here for consistentcy. (Let's talk about whether definitions should always get one later.